apps/web/src/app/(dashboard)/(management)/locatie/[location]/cohorten/[cohort]/(overview)/instructeurs/_components/instructors-table.tsx

```diff
@@ -49,11 +50,13 @@ export default function InstructorsTable({
   instructors,
   totalItems,
   locationId,
+  locationHandle,
 }: {
   cohortId: string;
   instructors: Awaited<ReturnType<typeof listInstructorsByCohortId>>;
   totalItems: number;
   locationId: string;
+  locationHandle: string;
```

Comment:
it's cleaner to use the `useParams()` hooks for this, as we do in other places :)